README.md

```diff
@@ -1939,19 +1939,21 @@ are very low.
 
 🄳 The catastrophic scenario where exercising ISOs triggers a large AMT bill, with no
 ability to sell the stock to pay taxes, is sometimes called the **AMT trap**. This
-[🔑infamous](https://medium.com/@barryjk/the-tax-law-that-is-unintentionally-hammering-silicon-valley-employees-894a7b54ba8a)
-[problem](https://www.nceo.org/articles/stock-options-alternative-minimum-tax-amt) has
+[infamous problem](https://www.nceo.org/articles/stock-options-alternative-minimum-tax-amt) has
 trapped many employees and
 [bankrupted people](http://blog.sfgate.com/dgreenberg/2012/06/22/tax-advice-from-the-dot-com-bubble-beware-of-isos/)
 during past dot-com busts.
 Now more people know about it, but it’s still a significant obstacle to plan around.
 
+📰new In 2017, Congress passed the Tax Cuts and Jobs Act (TCJA) which increases AMT exemptions and their phaseout thresholds. This means fewer people will be affected by AMT in 2018 than in [🔑prior years](https://medium.com/@barryjk/the-tax-law-that-is-unintentionally-hammering-silicon-valley-employees-894a7b54ba8a).
```

Comment:
- Don't put "new" (the text)
- Can we compare to other instances of TCJA mentions and make them consistent? We can safely link to Wikipedia on all of them for now and later pull that out as a definition or something.
- Run flowmark